src/main/webapp/assets/modules/post-contest/postContestChallengeDirective.js

```diff
@@ -1,4 +1,4 @@
-techlooper.directive("postContestChallenge", function ($http, utils) {
+techlooper.directive("postContestChallenge", function ($http, utils, jsonValue) {
```

Comment:
Line is too long.<br>'techlooper' is not defined.<br>'jsonValue' is defined but never used.<br>'utils' is defined but never used.<br>'$http' is defined but never used.